modules/library/src/main/scala/zio/elasticsearch/query/InnerHits.scala

```diff
@@ -41,12 +41,14 @@ private[elasticsearch] final case class InnerHits(
 }
 
 object InnerHits {
+  def empty: InnerHits = InnerHits(from = None, name = None, size = None)
+
   def from(value: Int): InnerHits =
-    InnerHits(from = Some(value))
+    InnerHits(from = Some(value), name = None, size = None)
 
-  def name(value: String): InnerHits =
-    InnerHits(name = Some(value))
+  def withName(value: String): InnerHits =
```

Comment:
Why is this changed? Not sorted.

Reply:
I think we are trying to avoid `with` prefix. @arnoldlacko?

Reply:
Maybe we could drop all these and just have `InnerHits.empty.name(...).from(...)`?